index.html

```diff
@@ -27,8 +27,21 @@
   <body>
     <div id="background"></div>
     <header>
+      <span class="header__sponsor-section">
+        Sponsored by 
+          <a href="https://www.madetech.com/">
+            <img 
+              src="./images/made-tech-logo-colour.png" 
+              class="header__made-tech-image"
+              alt="the logo of Made Tech: our VIP sponsor" />
+          </a>
+          ✰✰✰
```

Comment:
minor/question: 
for screen readers, I've read that emojis/ascii can be a bit annoying when read out so I've wrapped them in a hidden span `<span aria-hidden="true">✰✰✰</span>`